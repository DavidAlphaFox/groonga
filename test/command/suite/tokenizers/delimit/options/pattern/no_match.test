tokenize \
  'TokenDelimit("pattern", "nonexistent")' \
  "Hello"
