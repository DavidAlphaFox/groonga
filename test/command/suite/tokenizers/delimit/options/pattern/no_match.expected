tokenize   'TokenDelimit("pattern", "nonexistent")'   "Hello"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "Hello",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
